rest/parse.go

```diff
@@ -0,0 +1,33 @@
+package rest
+
+import (
+	"net/url"
+	"time"
+
+	"github.com/evergreen-ci/cedar/util"
+	"github.com/pkg/errors"
+)
+
+func parseTimeRange(vals url.Values, start, end string) (util.TimeRange, error) {
+	tr := util.TimeRange{EndAt: time.Now()}
```

Comment:
why now?

Reply:
Default time range is 0 time to current time. Line 29 below changes it if endAt is not empty.